src/Notifications/Channels/Discord/DiscordMessage.php

```diff
@@ -28,9 +28,11 @@ class DiscordMessage
 
     protected string $url = '';
 
-    public function from(string $username, string $avatarUrl = null): self
+    public function from(string $username = null, string $avatarUrl = null): self
     {
-        $this->username = $username;
+        if (! empty(trim($username))) {
```

Comment:
Let's revert all of the changes in this file. 

Add a check if the username is equal to an empty string, and if so, set the username to Laravel Backup